src/galaxy/StarSystemGenerator.cpp

```diff
@@ -1290,13 +1290,9 @@ SystemBody* StarSystemRandomGenerator::MakeGravPointForBodies(RefCountedPtr<Star
 	return grav;
 }
 
-std::string StarSystemRandomGenerator::GetStarNameFromId(const int id)
+char StarSystemRandomGenerator::GetStarSuffixFromOffset(const int offsetId) const
 {
-	// UGLY HACK
-	assert((id >= 0) && (id < 26));
-	char buf[16];
-	snprintf(buf, sizeof(buf), "%c", 'A' + id);
-	return buf;
+	return 'A' + Clamp(offsetId, 0, 26);
```

Comment:
is 26 included?

'A' + 26 = '['

Reply:
hmm, should probably be 25, assuming we'll be having systems with >25 stars

Reply:
damn, yeah it's inclusive clamp, i'll change it